boards/px4/raspberrypi/cmake/upload.cmake

```diff
@@ -0,0 +1,6 @@
+add_custom_target(upload
+        COMMAND rsync -arh --progress ${CMAKE_RUNTIME_OUTPUT_DIRECTORY} ${PX4_SOURCE_DIR}/posix-configs/rpi/*.config ${PX4_SOURCE_DIR}/ROMFS pi@"$ENV{AUTOPILOT_HOST}":/home/pi/px4
```

Comment:
So this uploads the configs. Is the px4 binary already uploaded?

Reply:
> 
> 
> So this uploads the configs. Is the px4 binary already uploaded?

${CMAKE_RUNTIME_OUTPUT_DIRECTORY} This one should contains binary executable part.